crypto/nrf_cc310_mbedcrypto/CHANGELOG.rst

```diff
@@ -9,6 +9,43 @@ Changelog - nrf_cc3xx_mbedcrypto
 
 All notable changes to this project are documented in this file.
 
+nrf_cc3xx_mbedcrypto - 0.9.16
+*****************************
+
+New version of the runtime library with the following bug fixes and improvements:
+
+* Built with MBEDTLS_PSA_CRYPTO_KEY_ID_ENCODES_OWNER enabled to ensure support with and without TF-M.
+* Added support for zero input message length for EDDSA for RFC test compliance.
+* Removed unused trace functions from runtime library.
```

Comment:
```suggestion
* Removed unused trace functions.
```

Reply:
Specifying runtime library sounds like it's being removed from one part of, or a variant of nrf_cc3xx_mbedcrypto. But by runtime library, one is referring to nrf_cc3xx_mbedcrypto, right?

Reply:
runtime library is already mentioned a few lines above, so the information can be removed. Will fix

Reply:
Fixed